spotify_player/src/event/mod.rs

```diff
@@ -27,7 +27,7 @@ pub enum PlayerRequest {
     Volume(u8),
     ToggleMute,
     TransferPlayback(String, bool),
-    StartPlayback(Playback),
+    StartPlayback(Playback, bool),
```

Comment:
should be `Option<bool>`

Reply:
Another approach is to set the shuffle state to true if this second parameter is true.